### PR TITLE
Removes the plugins that are neither used nor have a proper test environment 

### PR DESCRIPTION
**Pull request by wilderrodrigues**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,5 +54,9 @@ vmops.log*
 packaging/
 cosmic-packaging/
 
+<<<<<<< HEAD
```
Comment (remibergsma) [inline .gitignore]: @wilderrodrigues Looks like this file has an unresolved merge conflict?

Comment (wilderrodrigues) [inline .gitignore]: Holly cow! Missed that one. Thanks, @remibergsma !

**Comment by wilderrodrigues** [comment]
go build

**Comment by wilderrodrigues** [comment]
go build

**Pull request merged**